apps/docker-base/Makefile.docker-base

```diff
@@ -1,35 +1,40 @@
-# -*- makefile -*-
+# Makefile for Docker image setup
+
 help:
 	@echo "This should be run as part of the Dockerfile"
 	false
 
 user-setup:
+	# Create app group and user
 	groupadd --gid 1000 app
 	useradd -d /app --uid 1000 --gid app app
 	chown -R app:app /app
 
 apt-setup:
+	# Configure APT to keep downloaded packages
 	rm -f /etc/apt/apt.conf.d/docker-clean
 	echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
 
 apt-install:
-	DEBIAN_FRONTEND=noninteractive apt update
-	# gcc and python3-dev needed on arm for guidance
-	DEBIAN_FRONTEND=noninteractive apt -y install --no-install-recommends python3-poetry gcc python3-dev
+	# Install necessary packages without recommended packages
+	DEBIAN_FRONTEND=noninteractive apt update && \
```

Comment:
You don't need to do the && thing here.
make will run both of the steps and will abort if the previous one fails.